src/components/basicInformation/Section.tsx

```diff
@@ -0,0 +1,52 @@
+import SectionTitle from "@/components/basicInformation/utility/SectionTitle";
+import Image from "next/image";
+import { ReactNode } from "react";
```

Comment:
Minor TypeScript nitpick, but it'd be better to use `import type { ReactNode } from "react";` as you only use it for type annotations and not as a value. (Similar occurrences in other files - `src/pages/frequently-asked-questions.tsx` and `src/pages/what-is-domestic-violence.tsx`).